src/test/functional/pages/PersonalDetailsPage.ts

```diff
@@ -0,0 +1,72 @@
+import { expect } from '@playwright/test';
+import { Locator, Page } from 'playwright';
+
+const randomNumber = (Math.random() * 1000).toString().substring(0, 4);
+
+const nationality = 'Algerian',
+  nickname = 'NicknameTest',
+  maritalStatus = 'Married',
+  gender = 'Female',
+  smoker = 'Yes',
+  bloodType = 'A+';
+
+export class PersonalDetailsPage {
+  readonly page: Page;
+  readonly pageHeader: Locator;
+  readonly selectDropdown: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.pageHeader = page.getByRole('heading', { name: 'Personal Details' });
+    this.selectDropdown = page.getByText('-- Select --');
+  }
+
+  public async personalDetailsPageIsOpened(): Promise<void> {
```

Comment:
this seems to be an assertion, let's think about what prefix should we add in each assertion and stick to that naming: my suggestion is assertIfPersonalDetailsPageIsOpened or verifyIfPersonalDetailsPageIsOpened
the second thing that I would practice is to put down the assertion function at the end of the file --> at the bottom in order to have it grouped so at the top we have actions function and at the bottom assertion, it will be easier to debug code in the future.